gendocs/build.py

```diff
@@ -34,10 +36,9 @@ def gen_walkthrough_doc():
 
 def install_pydocmd():
     try:
-        pydocmd_dist = pkg_resources.get_distribution('pydoc-markdown')  # install name, not module name
-        version = pydocmd_dist.version
-    except pkg_resources.DistributionNotFound:
-        version = ''
+        version = metadata.version('pydoc-markdown')  # install name, not module name
+    except metadata.PackageNotFoundError:
+        version = None
```

Comment:
This will break the `version.endswith` below. Change it back to `None`, or change the check?

Reply:
done, check without whitespace changes: https://github.com/quiltdata/quilt/pull/2994/files?w=1